test/e2e_node/e2e_service.go

```diff
@@ -140,7 +140,11 @@ func (es *e2eService) startKubeletServer() (*exec.Cmd, error) {
 		"--api-servers", "http://127.0.0.1:8080",
 		"--address", "0.0.0.0",
 		"--port", "10250",
-		"--hostname-override", es.nodeName) // Required because hostname is inconsistent across hosts
+		"--hostname-override", es.nodeName, // Required because hostname is inconsistent across hosts
+		"--cgroup-root", "/",
+		"--system-cgroups", "/system",
```

Comment:
Can we do a check for systemd and set flags appropriately here?  Otherwise I will be opening an alternate PR tomorrow?  Maybe I am missing something.